app/org/sagebionetworks/bridge/services/AuthenticationServiceImpl.java

```diff
@@ -210,8 +213,12 @@ public void requestResetPassword(Study study, Email email) throws BridgeServiceE
         checkNotNull(email);
         
         Validate.entityThrowingException(emailValidator, email);
-
-        accountDao.requestResetPassword(study, email);
+        try {
+            accountDao.requestResetPassword(study, email);    
+        } catch(EntityNotFoundException e) {
+            // Suppress this. Otherwise it reveals if the account does not exist
+            logger.info("Request reset password attempt using address: "+study.getIdentifier()+" "+email.getEmail());
```

Comment:
Same here.

Also, if the user does a request reset password for an account that doesn't exist, should we trigger signup? Or is that impractical because signup needs a password?